services/console/src/components/navbar/theme/ThemeToggle.astro

```diff
@@ -59,8 +59,11 @@ import { THEME_TOGGLE_ID, ThemeId } from "./theme";
   }
 
   const initTheme = () => {
-    const theme = getTheme();
-    setTheme(theme);
+    // Only set the theme if the user has a preference for one.
+    const theme = localStorage.getItem(THEME_KEY);
+    if (theme) {
+      setTheme(theme);
+    }
```

Comment:
This lets the website theme automatically update when a user's system theme changes, while still respecting users' theme preferences that they set manually.